sample/src/main/java/com/fueled/flowr/sample/HomeFragment.java

```diff
@@ -57,4 +67,19 @@ private void displayLinkFragment() {
         getFlowr().open("/hello")
                 .displayFragment();
     }
+
+    private void displayFirstFragment() {
+        backStackIdentifier = getFlowr().open("/first")
+                .targetFragmentId(getFragmentId())
```

Comment:
and I think it's the `sourceFragmentId` not `target` 🤔

Reply:
`targetFragmentId` is inspired from `fragment.setTargetFragment(fragment, requestCode)` which specifies the target fragment for results. As per docs `@param fragment The fragment that is the target of this one.`